### PR TITLE
Feature/report sponsor

### PR DESCRIPTION
**Pull request by fdurand**
For review

### CODE REVIEW

**Comment by obilodeau** [comment]
This pull request contains the ipset changes along with the report ones. Please separate them.

Something like:

```
git branch -m feature/report_sponsor tmp-branch
git checkout stable
checkout -b feature/report_sponsor
git cherry-pick 838d567
git push origin feature/report_sponsor --force
```

should do it.

Une fois que t'es garanti que tes changements sont sur le serveur: `git branch -d tmp-branch`.

Pourquoi c'est arrivé? parce que t'as créer ta branche de feature/ipset et non de stable. `git checkout stable; git checkout -b feature/report-sponsor` t'aurais éviter ça.

Editorial notes: I just realized I switched language during the writing. Sorry about that ;)

**Comment by obilodeau** [comment]
thanks for the update!

**Comment by obilodeau** [comment]
I think `sponsoruser` is a misleading name. Is it users that were sponsored or sponsor themselves?

I would replace all mentions of `sponsor user` to `sponsors` and rename the report to `topsponsor`.

File: conf/ui.conf [hunk 1]
```diff
@@ -75,6 +75,7 @@ connectiontypereg=Connection-Type (Registered)
 ssid=SSID
 osclassbandwidth=Bandwidth per OS Class
 nodebandwidth=Top Bandwidth Consumers
+sponsoruser=Top Sponsor Users
```
Comment (obilodeau) [inline conf/ui.conf]: topsponsor=Top Sponsors

File: html/admin/status/reports.php [hunk 2]
```diff
@@ -76,6 +76,9 @@
       case 'nodebandwidth':
           $additional = 'Top 25 bandwidth consumers';
           break;
+      case 'sponsoruser':
+          $additional = 'Top 25 sponsor users';
```
Comment (obilodeau) [inline html/admin/status/reports.php]: Top 25 Sponsors

File: lib/pf/pfcmd/report.pm [hunk 3]
```diff
@@ -844,6 +860,8 @@ Olivier Bilodeau <obilodeau@inverse.ca>
 
 Francois Gaudreault <fgaudreault@inverse.ca>
 
+fabrice Durand <fdurand@inverse.ca>
```
Comment (obilodeau) [inline lib/pf/pfcmd/report.pm]: s/fabrice/Fabrice/

**Comment by fgaudreault** [comment]
Please review one last time.  If OK, please merge :)

**Pull request merged**